aptos-move/framework/src/module_metadata.rs

```diff
@@ -63,6 +63,9 @@ pub struct KnownAttribute {
 /// Enumeration of known attributes
 #[derive(Debug, Clone, PartialEq, Eq, PartialOrd, Ord, Serialize, Deserialize)]
 pub enum KnownAttributeKind {
+    // An older compiler placed view functions at 0. This was then published to
```

Comment:
Wow. Was this because enum numbering starts at 0? You should have seen this before by failing `./api` tests.

Reply:
Scratch the failure, indeed no failure unless comparison test